antora.yml

```diff
@@ -3,3 +3,7 @@ title: Mule Runtime
 version: '4.1'
 nav:
 - modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    endOfLife: true
```

Comment:
Looks like extended support ended for v4.1 in Nov 2022. So, let's make sure we want to say EOL here before merging, and if so, merge. Need to determine when we can archive docs.